sample/app/src/main/java/io/walletconnect/example/MainActivity.kt

```diff
@@ -86,6 +92,7 @@ class MainActivity : Activity(), Session.Callback {
                     ::handleResponse
             )
             this.txRequest = txRequest
+            navigateToWallet()
```

Comment:
This will send the same `?bridge=xxx&key=xxx` this might be a problem (Doesn't seem to be on metamask), only way I get it to deeplink after the app has sent the transaction.